wisegui/src/lib.rs

```diff
@@ -236,6 +252,60 @@ impl<'a> Painter<'a> {
         }
     }
 
+    pub fn line(&mut self, start: (i32, i32), end: (i32, i32), stroke: Color) {
```

Comment:
Looks good so far, nice and simple :) as laid out [here](https://github.com/yupferris/wisegui/issues/1#issuecomment-339253151) I won't accept this PR until I can license this repo, and I think this impl should also clip to the bounds of the screen. In the future the public API will probably expose a clipping rectangle (allowing us to more easily implement scrolling views for example), but for now clipping to the screen bounds will be sufficient and easily adaptable later.

Reply:
What do you think about adding an Optional<(i32, i32, i32, i32)> that if None default to the screen rect? Do you want to keep using tuples for all the coordinates for now or add a couple simple structs (Point, Rect)?